rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -812,14 +813,14 @@ protected BlockValidator buildBlockValidator() {
     }
 
     protected GenesisLoader buildGenesisLoader() {
-        RskSystemProperties rskSystemProperties = getRskSystemProperties();
-        ActivationConfig.ForBlock genesisActivations = rskSystemProperties.getActivationConfig().forBlock(0L);
+        RskSystemProperties systemProperties = getRskSystemProperties();
```

Comment:
SonarCloud pointed this out on this PR, probably because I changed some code into RskContext.java. I didn't want to leave more code smells than before my PR and that is why this change is here.
It could be in another PR but at the same time is not so major to step in the way of reviewing this PR. It will stay in this PR.